build.gradle

```diff
@@ -2,7 +2,7 @@ plugins {
   id 'java'
   id 'application'
   id 'com.github.johnrengelman.shadow' version '2.0.4'
-  id "com.diffplug.gradle.spotless" version '3.14.0'
+  id "com.diffplug.gradle.spotless" version '3.15.0'
```

Comment:
Seems super weird that the old version went away -- should it be in maven? wondering if we can ask gradle to retry pulling dependencies a couple of times before failing entirely

Reply:
So here's 3.14 in the gradle repo: https://plugins.gradle.org/plugin/com.diffplug.gradle.spotless/3.14.0

Technically we should not have had to do a version bump here.